transpiler/src/main/java/org/jsweet/transpiler/Java2TypeScriptTranslator.java

```diff
@@ -685,6 +679,23 @@ private boolean isMappedOrErasedType(Symbol symbol) {
 				|| context.hasAnnotationType(symbol, JSweetConfig.ANNOTATION_ERASED);
 	}
 
+	/**
+	 * Ensures that the module that corresponds to the given element is used (imported).
+	 */
+	public void ensureModuleIsUsed(Element element) {
```

Comment:
super ! Par contre je pense qu'il ne faudrait pas qu'elle soit publique. protected tout au plus

Reply:
It is a public utility method that should be accessible from the adapters... So let's keep it public. Although there is another issue in the API to get access to the printer's methods but that's a different issue that should be solved elsewhere.